assets/js/dashboard.js

```diff
@@ -79,49 +77,78 @@ function renderDashboardResults(searchQuery) {
         })
         .then(data => {
             console.log(data); // SEE: data format.
-            // Manage data here
+            let daysData = get5DayForecast(data.list);
+            display5DayForecast(daysData);
         })
         .catch(error => {
             console.log(error);
         });
 
     // API Calls to get location details
-    // API fetch request | "https://www.googleapis.com/customsearch/v1?key={API key}&cx={PSE ID}&q={city name}&searchType=image&num=1"
-    // This is for a Google SafeSearch.
-    let fetchURLGoogleDesc = googleAPIURL.concat("key=", googleAPIKeyAN, "&cx=", googlePSEID, "&q=", searchQuery, "&num=1")
-    fetch(fetchURLGoogleDesc)
-        .then(response => {
-            if (!response.ok) {
-                throw new Error("API response was not okay. (Google query)");
-            }
-            return response.json();
-        })
+    fetchWithRetry(googleAPIURL.concat("key=", googleAPIKeyAN, "&cx=", googlePSEID, "&q=", searchQuery, "&num=1"))
```

Comment:
fetchWithRetry is not an existing function, **we would need to make our own.** (May be outside the course content).
I think it is better to keep the old way but **it's okay to remove declaration of fetchURLGoogleDesc variable** and instead place the concatenation inside the function parameter.
**This currently does not work.**
"Uncaught ReferenceError: fetchWithRetry is not defined."